src/current/_includes/v23.2/ui/insights.md

```diff
@@ -24,54 +24,66 @@ The rows in this page are populated from the [`crdb_internal.transaction_content
-- **Index Name**: The name of the contended index.
+The transaction execution details view provides more information on a transaction execution insight.
+
+#### Default details
+
+This information is displayed for all insight types.
+
+Field/Column | Description
+------|------------
+Start Time | The timestamp when the transaction execution started.
+End Time | The timestamp when the transaction execution ended.
+Elapsed Time | The time that elapsed during transaction execution.
+CPU Time | The amount of CPU time spent executing the transaction. The CPU time represents the time spent and work done within SQL execution operators. {% if page.cloud != true -%}The CPU time includes time spent in the [SQL layer]({% link {{ page.version.version }}/architecture/sql-layer.md %}). It does not include time spent in the [storage layer]({% link {{ page.version.version }}/architecture/storage-layer.md %}).{% endif -%}{% if page.cloud == true %}The CPU time includes time spent in the [SQL layer](../stable/architecture/sql-layer.html). It does not include time spent in the [storage layer](../stable/architecture/storage-layer.html).{% endif %}
+Rows Read | The total number of rows read by the transaction execution.
+Rows Written | The total number of rows written by the transaction execution.
+Priority | The [priority]({{ link_prefix }}transactions.html#set-transaction-priority) of the transaction execution.
+Number of Retries | The total number of retries of the transaction.
+Session ID | The ID of the [session]({{ link_prefix }}ui-sessions-page.html) the transaction was executed from.
+Application | The name specified by the [`application_name` session setting]({{ link_prefix }}show-vars.html#supported-variables).
+Transaction Fingerprint ID | The fingerprint ID of the transaction execution. To view [details of the transaction]({{ link_prefix }}ui-transactions-page.html#transaction-details-page), click the fingerprint ID.
+Insights | The [insight type](#workload-insight-types).
+Details | Provides details on the insight. For example, if the insight type is High Contention, **Time Spent Waiting** and **Description** are displayed. If the insight type is Failed Execution, **Error Code** and **Error Message** are displayed.
+
+#### Conditional details
+
+This information is displayed for only the following insight types.
+
+1. [**High Contention**](#high-contention):
+All transaction executions flagged with this insight type will display a **Transaction with ID {transaction ID} waited on** section which provides details of the blocking transaction execution.
+1. [**Failed Execution**](#failed-execution):
+Certain transaction executions flagged with this insight type will display a **Failed Execution** section with **Conflicting Transaction** and **Conflicting Location** information. The following 3 conditions are required:
+  - The `sql.contention.record_serialization_conflicts.enabled` [cluster setting]({{ link_prefix }}cluster-settings.html) is set to `true`.
```

Comment:
Can we specify the default value here? @aadityasondhi, any reason why it's not true by default? 

If so, we should call that out.

Reply:
I am not sure.

@arulajmani do you have more context here?

Reply:
I made this setting. It's off by default because the team discussed it should be an opt-in feature initially to minimize any unforeseen issues. Do you think it should be on by default @kevin-v-ngo ?

Reply:
https://github.com/cockroachdb/cockroach/issues/114416

Reply:
specified that the default value is false.